std/assembly/atomics.ts

```diff
@@ -1,6 +1,7 @@
 import { ArrayBufferView } from "./arraybuffer";
 
 export namespace Atomics {
+
```

Comment:
This works as well: `git commit --allow-empty -m "The message"` :)

Reply:
That's good to know)